test/use_case/get_loadable_games/load_game/LoadGameInteractorTest.java

```diff
@@ -1,9 +1,10 @@
-package use_case.load_game;
+package use_case.get_loadable_games.load_game;
```

Comment:
It looks like you accidentally moved the load_game folder in test/use_case to be under get_loadable_games. If you could move it back to be at the same level as the others that would be great.